jobs/haproxy/spec

```diff
@@ -74,6 +74,16 @@ properties:
     description: "Timeout for requests queued waiting for free connection slots (in seconds)"
     default:     30
 
+  ha_proxy.stats_enable:
+    description: "If true, haproxy will enable a socket for stats. You can see the stats on haproxy_ip:9000/haproxy_stats"
+    default: false
+  ha_proxy.stats_user:
+    description: "User name to authenticate haproxy stats"
+    default: admin
+  ha_proxy.stats_password:
+    description: "Password to authenticate haproxy stats"
+    default: admin
+
```

Comment:
I think I would prefer to not have default user/passwords in the specfile, to force people to provide their own.